agent/bench-scripts/postprocess/fio-postprocess-viz.py

```diff
@@ -1,12 +1,8 @@
-#!/usr/bin/env python2
-from __future__ import print_function
+#!/usr/bin/env python3
 
 import sys
 from os.path import join, basename
-try:
-    from configparser import SafeConfigParser
-except ImportError:
-    from ConfigParser import SafeConfigParser
+from configparser import SafeConfigParser
```

Comment:
Running the unit tests on F29, I get a warning from this line (in this and every other file that contains it, so many unit tests fail) - here's a typical example:
````
  +++ Running test-31 pbench-copy-results --prefix=foo/bar --user=ndk
+ /var/tmp/pbench-test-utils/opt/pbench-agent/util-scripts/pbench-add-metalog-option:20: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
+   config = SafeConfigParser()
+ /var/tmp/pbench-test-utils/opt/pbench-agent/util-scripts/pbench-add-metalog-option:20: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
+   config = SafeConfigParser()
  tar --create --force-local "pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32" | xz -T0 > "/var/tmp/pbench-test-utils/pbench/tmp/pbench-copy-results.NNNNN/testhost/pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32.tar.xz" 
  --- Finished test-31 pbench-copy-results (status=0)

````